clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -360,57 +371,68 @@ public void testNoCoordinator() {
     @ParameterizedTest
     @ApiKeyVersionsSource(apiKey = ApiKeys.CONSUMER_GROUP_HEARTBEAT)
     public void testValidateConsumerGroupHeartbeatRequest(final short version) {
+        heartbeatState = new HeartbeatState(
```

Comment:
I guess those creations can be moved to a helper to be shared by `testValidateConsumerGroupHeartbeatRequestAssignmentSentWhenLocalEpochChanges`, `testValidateConsumerGroupHeartbeatRequest` and `testFirstHeartbeatIncludesRequiredInfoToJoinGroupAndGetAssignments`

Reply:
Good point! I have moved those object creations into a helper method now. Let me know what you think.